### DISCUSSION TITLE
Math rendering

### DISCUSSION THREAD

**Discussion by MatthewCaseres** [markdoc/markdoc#108] [Q&A] [answered]
Can we render math? I couldn't find it in the docs.

**Comment by mfix-stripe** [comment, marked as answer]
@MatthewCaseres at the moment, GitHub-flavored math rendering is not supported by Markdoc, as our syntax is a superset of the [CommonMark spec](https://spec.commonmark.org/0.30/).

You can easily support this in your own documents by creating a `{% math %}` tag.

**Reply by MatthewCaseres** [reply]
Wow, when you said "GitHub-flavored math" I found that GitHub now supports math. $nice$

**Reply by AustenLamacraft** [reply]
It's a bit of a shame not to be able to use the usual LaTeX delimiters `\(...\)` and `\[...\]`, or the TeX versions `$...$` and `$$...$$`, as these are so widely adopted. For example, writing big chunks of math would be challenging without the possibility of a preview.
